app/[locale]/(interactive)/4-15/page.tsx

```diff
@@ -0,0 +1,61 @@
+"use client";
+import NextButton from "@/components/ui/nextButton";
+import { useShareYourselfWordsStore } from "@/stores/ShareYourselfWords.store";
+import { motion } from "framer-motion";
+import { useTranslations } from "next-intl";
+import Link from "next/link";
+
+type Props = {};
+
+const Page = (props: Props) => {
+  const t = useTranslations("4-15");
+
+  const words = useShareYourselfWordsStore(
+    (state: { words: string }) => state.words,
+  );
+  const updateWords = useShareYourselfWordsStore(
+    (state: { updateWords: (words: string) => void }) => state.updateWords,
+  );
+
```

Comment:
this can be reduced to const { words, updateWords } = useShareYourselfWordStore()